modules/BuildUtilities/src/server/python/zzz-config-to-envs.py

```diff
@@ -0,0 +1,146 @@
+# Process configs
+import os 
+from dotenv import load_dotenv
+from pathlib import Path
+import pandas as pd
+import json
+import argparse
+
+
+dotenv_path = Path('./conf/docker/acas/environment/.env')
+load_dotenv(dotenv_path=dotenv_path)
+
+dotenv_path = Path('.env')
+load_dotenv(dotenv_path=dotenv_path)
+
+def get_text_file_lines(file_path):
+    with open(file_path, 'r') as f:
+        lines = f.readlines()
+    return lines
+
+def write_lines_to_file(file_path, lines):
+    with open(file_path, 'w') as f:
+        for line in lines:
+            f.write(line)
+
+def process_property_lines(lines, example_names=None):
```

Comment:
Could you add a docstring explaining what `example_names` does? I can sortof get it (only looking for novel configs) but I'm still confused.